docker-compose.yml

```
@@ -406,8 +406,8 @@ services:
       Cache__Enabled: true
       EnableIntegrationFeature: true
       ProblemDetails__IncludeExceptionDetails: false
-      LinkTokenService__EnableTokenGenerationEndpoint: true
-      LinkTokenService__Authority:      
+      LinkTokenService__EnableTokenGenerationEndpoint: false
+      LinkTokenService__Authority: placeholder     
```

yamllint

[error] trailing spaces

(trailing-spaces)